src/app/components/newproposal/FundingInput.tsx

```diff
@@ -1,28 +1,22 @@
 import React, { useCallback, useEffect, useState } from 'react';
 
-import BigNumber from 'bignumber.js';
 import styled from 'styled-components';
 
 import AddressInputPanel from 'app/components/AddressInputPanel';
 import CurrencyInputPanel from 'app/components/CurrencyInputPanel';
 import { BoxPanel } from 'app/components/newproposal/RatioInput';
-import { PROPOSAL_CONFIG, CURRENCY_LIST } from 'app/containers/NewProposalPage/constant';
+import { PROPOSAL_CONFIG } from 'app/containers/NewProposalPage/constant';
+import { SUPPORTED_TOKENS_LIST } from 'constants/tokens';
 import { getTokenFromCurrencyKey } from 'types/adapter';
-import { Currency } from 'types/balanced-sdk-core';
+import { Currency, CurrencyAmount } from 'types/balanced-sdk-core';
 
+type Amount = {
```

Comment:
@octocat512 I've used Amount data structure because I think the empty value should show a placeholder instead of 0 

```
 CurrencyAmount.fromRawAmount(fundingRes[0].currency, '').toFixed(2) => 0.00
```
Please check it again. Thank you!

![image](https://user-images.githubusercontent.com/17588370/147763018-09c4059e-6895-4882-b92f-41441e646645.png)

Reply:
> @octocat512 I've used Amount data structure because I think the empty value should show a placeholder instead of 0
> 
> ```
>  CurrencyAmount.fromRawAmount(fundingRes[0].currency, '').toFixed(2) => 0.00
> ```
> 
> Please check it again. Thank you!
> 
> ![image](https://user-images.githubusercontent.com/17588370/147763018-09c4059e-6895-4882-b92f-41441e646645.png)

You can use `undefined` for the empty input.